HelpSource/Classes/Buffer.schelp

```diff
@@ -88,7 +88,7 @@ The number of channels for each buffer. The default is 1.
 argument:: completionMessage
 A valid OSC message or a Function which will return one. A Function will be passed each Buffer and its index in the array as arguments when evaluated.
 argument:: bufnum
-An explicitly specified buffer number for the initial buffer. Generally this is not needed.
+An explicitly specified buffer number for the initial buffer. Use of this argument is highly discouraged, and unnecessary under normal usage (see link::Classes/Buffer#*alloc::).
```

Comment:
could you please also change this wording as you did in the above? ("highly discouraged" is not really what we want to say, we really just should explain what it means).

(same below)